Cloudflare Deployer fails, cannot find entrypoint index.mjs

When running mastra deploy with the Cloudflare deployer, it fails to find the index.mjs entrypoint. Others in the Discord have also been experiencing this issue.

```
    "@mastra/core": "^0.4.4",
    "@mastra/deployer-cloudflare": "^0.1.7",
    "mastra": "^0.2.8",
```

```
INFO [2025-03-02 14:25:59.929 -0600] (DEPLOYER - CLOUDFLARE): Done in 8.3s using pnpm v10.4.1
INFO [2025-03-02 14:25:59.963 -0600] (DEPLOYER - CLOUDFLARE): Done installing dependencies
⛅️ wrangler 3.111.0
--------------------
INFO [2025-03-02 14:26:01.410 -0600] (DEPLOYER - CLOUDFLARE): ⛅️ wrangler 3.111.0
--------------------
✘ [ERROR] The entry-point file at "index.mjs" was not found.
INFO [2025-03-02 14:26:01.704 -0600] (DEPLOYER - CLOUDFLARE): ✘ [ERROR] The entry-point file at "index.mjs" was not found.
```

Here's my index.ts file:
```
export const mastra = new Mastra({
  ...,
  deployer: new CloudflareDeployer({
    auth: {
      apiToken: String(process.env.CLOUDFLARE_API_KEY),
    },
    scope: String(process.env.CLOUDFLARE_ACCOUNT_ID),
    projectName: 'my-project-name',
  }),
});
```

Upon investigating the wrangler.json file in the .mastra build ouput, it points to `"main":"index.mjs"` but this does not exist in that directory.

<img width="301" alt="Image" src="https://github.com/user-attachments/assets/f055373f-d322-404b-9c67-66781e5de3df" />

--- a/.changeset/lucky-taxis-feel.md
+++ b/.changeset/lucky-taxis-feel.md
@@ -1,0 +1,5 @@
+---
+'@mastra/deployer-cloudflare': patch
+---
+
+have cloudflare wrangler point to correct entry point file

--- a/deployers/cloudflare/src/index.ts
+++ b/deployers/cloudflare/src/index.ts
@@ -59,7 +59,7 @@ export class CloudflareDeployer extends Deployer {
 
     const wranglerConfig: Record<string, any> = {
       name: cfWorkerName,
-      main: 'index.mjs',
+      main: './output/index.mjs',
       compatibility_date: '2024-12-02',
       compatibility_flags: ['nodejs_compat'],
       observability: {

